src/main/java/com/github/maxopoly/finale/listeners/PlayerListener.java

```diff
@@ -123,6 +123,10 @@ public void playerLogin(PlayerJoinEvent e) {
 			// Register login for custom health regen
 			manager.getPassiveRegenHandler().registerPlayer(e.getPlayer().getUniqueId());
 		}
+		if (manager.getCTPOnLogin()) {
```

Comment:
also check if the combattag handler is null, it might be if CTP is not on the server in which case this would throw an exception on every login